welltestpy/process/processlib.py

```diff
@@ -40,9 +40,7 @@ def normpumptest(pumptest, pumpingrate=-1.0, factor=1.0):
     pumptest.pumpingrate = pumpingrate
 
     for obs in pumptest.observations:
-        pumptest.observations[obs].observation *= (
-            factor * pumptest.rate / oldprate
-        )
+        pumptest.observations[obs].observation *= factor * pumptest.rate / oldprate
```

Comment:
Result of the black reformatting done. I dont know if it worked correctly :)

Reply:
I guess you didn't use `black -l 79 .` where `-l 79` bounds the line-length to 79 characters.